pipelines/folioRancher/folioTestingTools/folioSchemaCompare.groovy

```diff
@@ -0,0 +1,221 @@
+#!groovy
+import hudson.util.Secret
+import org.jenkinsci.plugins.workflow.libs.Library
+import java.time.*
+import org.folio.rest.GitHubUtility
+import org.folio.Constants
+import groovy.json.JsonSlurperClassic
+
+@Library('pipelines-shared-library@RANCHER-838') _
+
+String getOkapiVersion(folio_repository, folio_branch) {
+    def installJson = new URL('https://raw.githubusercontent.com/folio-org/' + folio_repository + '/' + folio_branch + '/install.json').openConnection()
+    if (installJson.getResponseCode().equals(200)) {
+        String okapi = new JsonSlurperClassic().parseText(installJson.getInputStream().getText())*.id.find{it ==~ /okapi-.*/}
+        if(okapi){
+            return okapi - 'okapi-'
+        } else {
+            error("Can't get okapi version from install.json in ${folio_branch} branch of ${folio_repository} repository!" )
+        }
+    }
+    error("There is no install.json in ${folio_branch} branch of ${folio_repository} repository!" )
+}
+
+properties([
+    buildDiscarder(logRotator(numToKeepStr: '20')),
+    disableConcurrentBuilds(),
+    parameters([
+        jobsParameters.refreshParameters(),
+        jobsParameters.repository(),
+        jobsParameters.branch('folio_repository', 'folio_branch_src'),
+        jobsParameters.branch('folio_repository', 'folio_branch_dst'),
+        string(name: 'backup_name', defaultValue: '', description: '(Optional) RDS snapshot name. If empty create env from scratch', trim: true),
+        string(name: 'slackChannel', defaultValue: '', description: 'Slack channel name where send report (without #)', trim: true)])])
+
+def rancher_cluster_name = 'folio-perf'
+def rancher_project_name = 'data-migration'
+def config_type = 'performance'
+def tenant_id
+def admin_username
+def admin_password
+def tenant_id_clean ='clean'
+def diff = [:]
+def resultMap = [:]
+def pgadminURL = "https://$rancher_cluster_name-$rancher_project_name-pgadmin.ci.folio.org/"
+def okapiVersion = getOkapiVersion(params.folio_repository, params.folio_branch_src)
+
+ansiColor('xterm') {
+    if (params.refresh_parameters) {
+        currentBuild.result = 'ABORTED'
+        println('REFRESH JOB PARAMETERS!')
+        return
+    }
+    node('rancher') {
+        try {
+            stage('Init') {
+                currentBuild.result = 'SUCCESS'
```

Comment:
Is this assignment needed?

Reply:
yes Dear Oleksandr
we need to assign to the relevant team , based on schema diff
f.e 
![image](https://github.com/folio-org/pipelines-shared-library/assets/138441317/7c6037c4-f2cb-44ed-a401-3101f8997167)